front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -315,13 +314,15 @@ class PBExpenses extends PureComponent<Props & InjectedIntlProps, State> {
       let progressBarColor: 'green' | 'red' | '' = '';
 
       if (participationContextType === 'project' && !isNilOrError(project)) {
-        totalBudget = project.attributes.max_budget as number;
+        minBudget = project.attributes.min_budget as number;
```

Comment:
Because of a component that's not ideally designed. It's trying to handle two cases (projects and phases). Given the checks that are done, you can be certain there is going to be a min_budget (which is of type number), but without the casting, TS says it can still be undefined. Hence the casting. Adding it to the debt list for this epic as well.

(We should start typing differentiating project types more specifically (differentiatie between timeline and continuous projects in the types) to deal with this problem -- you guessed it, it's on my debt list for this epic and previous ones.)

Reply:
Hm, indeed! 
Casting like this is a bit dangerous and negates the whole point of using typescript so my personal preference would be to add a check for undefined (until we can fix the typings) here even if it's not strictly necessary. The only reason is that more checks could not conceivably lead to bugs but forcing the type can.
Not a blocking issue though so I leave it up to you.

Reply:
Agree, just changed it!